### PR TITLE
0.9.9 known issues - quick list

### PR DESCRIPTION
**Pull request by duddun**

### CODE REVIEW

**Comment by duddun** [comment]
this is just a preliminary pass at this -- there is editing to do

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 1]
```diff
@@ -157,3 +157,14 @@ Bugs
...
 
+Known Issues
+^^^^^^^^^^^^
+
+  * Provisioner: during chef-solo, between bootstrap and coopr hosts running, non-fatal sudo error
+  * UI: If base is removed from selected roles/services to install, it still gets installed
+  * UI: Unable to edit provider
+  * Node usage API: Coopr Node Usage Hours 'now-Nx' does not work               `COOPR-714 <https://issues.cask.co/browse/COOPR-714>`_
+  * Node usage API: API allows arbitrary, random parameters to be used as keys  `COOPR-716 <https://issues.cask.co/browse/COOPR-716>`_
+  * CDH install fails when distribution_version >= 5.3 specified                `COOPR-740 <https://issues.cask.co/browse/COOPR-740>`_
```
Comment (dereklwood) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: we are fixing this

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 2]
```diff
@@ -157,3 +157,14 @@ Bugs
...
+  * Template Creation. Service Constraints min/max boxes are uneditable         `COOPR-739 <https://issues.cask.co/browse/COOPR-739>`_
+  * Node deletion issue (Joyent)
```
Comment (dereklwood) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: we should be more specific.  maybe this warrants a jira?

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 3]
```diff
@@ -157,3 +157,15 @@ Bugs
...
   * `COOPR-731 <https://issues.cask.co/browse/COOPR-731>`_ - Standalone startup script broken
 
+
+Known Issues
+------------
+
+  * Provisioner: plugins must now extend from Coopr::Plugin::Automator or Coopr::Plugin::Provider.  Existing custom plugins need to be updated accordingly for 0.9.9
+  * Provisioner: non-fatal sudo errors may be seen prior to coopr_hosts (part of base service) being run
+  * UI: If base is removed from selected roles/services to install, it still gets installed
+  * UI: Unable to edit provider
```
Comment (dereklwood) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: needs more details, isn't there a jira?

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 4]
```diff
@@ -157,3 +157,15 @@ Bugs
...
+  * Node usage API: Coopr Node Usage Hours 'now-Nx' does not work                     `COOPR-714 <https://issues.cask.co/browse/COOPR-714>`_
+  * Node usage API: API allows arbitrary, random parameters to be used as keys        `COOPR-716 <https://issues.cask.co/browse/COOPR-716>`_
+  * Service Constraints fields do not display entered value during Template creation  `COOPR-739 <https://issues.cask.co/browse/COOPR-739>`_
+  * Node deletion issue (Joyent)
```
Comment (dereklwood) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: needs more details, and a jira

File: coopr-docs/docs/source/release-notes/release-0.9.9.rst [hunk 5]
```diff
@@ -157,3 +157,15 @@ Bugs
   * `COOPR-730 <https://issues.cask.co/browse/COOPR-730>`_ - joyent plugin provider fails on CentOS: confirm stage fails to unmount disk
   * `COOPR-731 <https://issues.cask.co/browse/COOPR-731>`_ - Standalone startup script broken
 
+
+Known Issues
+------------
+
+  * Provisioner: plugins must now extend from Coopr::Plugin::Automator or Coopr::Plugin::Provider.  Existing custom plugins need to be updated accordingly for 0.9.9
+  * Provisioner: non-fatal sudo errors may be seen prior to coopr_hosts (part of base service) being run
+  * UI: If base is removed from selected roles/services to install, it still gets installed
```
Comment (awholegunch) [inline coopr-docs/docs/source/release-notes/release-0.9.9.rst]: "prior to install" ??

**Comment by wolf31o2** [comment]
:+1: LGTM

**Comment by awholegunch** [comment]
LGTM

**Pull request merged**